App/Store/StoreManager.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace App\Store;
+
+use App\{Enums\Status, Interface\FileManagerInterface, Model\Collection, Model\Fast};
+use DateTime;
+use Exception;
+
+
+class StoreManager implements FileManagerInterface
+{
+
+    protected string $file = "./store.json";
+
+    /**
+     * Fetches all fasts from store.json if any and returns a collection
+     * @return Collection
+     * @throws Exception
+     */
+    public function getAll(): Collection
+    {
+        $today = new DateTime('NOW');
+        $fastArray = [];
+        $storeFasts = json_decode(
```

Comment:
Maybe create a class helpers for methods like this
so you can do

Json::decode()
Json::encode()